src/index.html

```diff
@@ -1,12 +1,319 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a href="#" class="nav__link">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="product">
+        <div class="product__card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            class="product__photo"
+            alt="imac"
+          >
+          <h2 class="product__card-name">
+            Моноблок APPLE A1419 iMac<br> 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="product__card-code">
+            Код товара: 195434
+          </p>
+          <div class="product__grade">
+            <div class="stars">
+              <div class="star star--1"></div>
```

Comment:
name according to BEM

```suggestion
            <div class="stars">
              <div class="stars__star stars__star--1"></div>
```